jenkins/gradle/gradle-check.jenkinsfile

```diff
@@ -60,6 +60,7 @@ pipeline {
         JAVA11_HOME="/var/jenkins/tools/hudson.model.JDK/openjdk-11"
         JAVA14_HOME="/var/jenkins/tools/hudson.model.JDK/openjdk-14"
         JAVA17_HOME="/var/jenkins/tools/hudson.model.JDK/openjdk-17"
+        JAVA19_HOME="/var/jenkins/tools/hudson.model.JDK/openjdk-19"
```

Comment:
@peterzhuamazon need you help here to preconfigure JDK-19 in Jenkins tooling, thank you